test/demo-logger-using-pvc/README.md

```diff
@@ -0,0 +1,52 @@
+# Containerized-Distributed-Logger 📝
+Logging Service to demonstrate the distributed logging with log collector.
+***
+
+### ⚽ Current Functionality:
```

Comment:
Simply too many codacy warnings, need to address even if its test code. Address until conflicting ones seen

Reply:
`add 2 spaces` & `remove 2 spaces` are conflicting and the only remaining codacy issue now.

Reply:
Ok